### ISSUE TITLE
Promote the Flatpak

### ISSUE THREAD

**Issue by introt** [zim-desktop-wiki/zim-desktop-wiki#2427]
**Is your feature request related to a problem? Please describe.**
https://zim-wiki.org/downloads.html doesn't mention the Flatpak which currently has around 20k installs on Flathub.

**Describe the solution you'd like**
Add a link to Flathub, mentioning that it's packaged by a third party and has it's own bug tracker for Flatpak-derived issues.

**Describe alternatives you've considered**
Keeping quiet about one of the most reproducible installation methods we have on Linux?

**Additional context**
The Flatpak is maintained by @cheese, a long-time contributor and a former maintainer of Zim's Fedora package.

- Flathub link: https://flathub.org/apps/org.zim_wiki.Zim
- Manifest: https://github.com/flathub/org.zim_wiki.Zim

**Comment by Nick222** [comment]
https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/2416

**Comment by introt** [comment]
Agreed, the Flatpak's limitations are also worth a mention. Given their nature there might be a general article on flatpaks' limitations on the internet we could link to, though "be aware that the sandboxing may interfere with plugin functionality" imo conveys the message just as well.
